AWS-CodePipeline/sam-monorepo-pipeline-template/questions.json

```diff
@@ -0,0 +1,234 @@
+{
+    "questions": [
+        {
+            "key": "monorepo_project_subfolder",
+            "question": "What is subfolder path of your project from the root of the repository?",
+            "isRequired": true
+        },
+        {
+            "key": "monorepo_ssm_prefix",
+            "question": "What is the SSM Prefix used to store metadata about this Monorepo?",
```

Comment:
```suggestion
            "question": "What is the SSM prefix used to store metadata about this monorepo?",
```

Reply:
Also what's an "SSM Prefix"? Googling it shows no helpful results.

Reply:
SSM Parameter names have to be unique in each account/region, but they can be [organised in a path structure](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-hierarchies.html). This is the "path prefix" to the SSM parameters, used by the Lambda function to store information between executions.